### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by ImperialSpaceCoalition**
fixed admin features

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+4. [Usage](#usage)
+5. [Admin Functions](#admin-functions)
+6. [Screenshots](#screenshots)
+7. [Contributing](#contributing)
+8. [License](#license)
+
+---
+
+## Introduction
+Welcome to the Whiskers README! This application is designed to help whiskey enthusiasts find, rate, and store their favorite whiskeys. As an added feature, administrators have the ability to manage the whiskey database by adding or deleting entries.
```
Comment (mbarrie1979) [inline README.md:16]: Remove "delete" (we can only add currently)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+---
+
+## Features
+- **User Features:**
+  - Search for whiskeys by name, type, or rating.
```
Comment (mbarrie1979) [inline README.md:22]: Only name search is available not type or rating currently

File: README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  - Rate whiskeys based on personal preference.
+  - Store favorite whiskeys for future reference.
+
+---
+
+## Installation
+To install the Whiskey Application, follow these steps:
+1. Clone the repository from GitHub: `git clone https://github.com/milindmjoshi/whiskers.git`
+2. Navigate to the project directory: `cd whiskers`
```
Comment (mbarrie1979) [inline README.md:31]: add 3. Run schema.sql to create whiskers_db database

File: README.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+## Usage
+Once the application is running, users can:
+- Search for whiskeys using the search functionality.
+- Rate whiskeys by clicking on the rating stars.
+- Store favorite whiskeys by adding them to their personal collection.
+
+
+## Admin Functions 
+To be added in future releases
```
Comment (mbarrie1979) [inline README.md:46]: Admin can upload a whiskey with name, description and photo

File: README.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+
+---
+
+## Screenshots
+![Login Page](./public/assets/images/login.png "Login Page")
+![Signup Page](./public/assets/images/sign-in.png "Signup Page")
+![Whiskey Feed](./public/assets/images/whiskey-feed.png "Whiskey Feed")
+![Add/delete Whiskey](screenshots/add-whiskey.png "Add Whiskey")
```
Comment (mbarrie1979) [inline README.md:55]: correct this path

**Comment by milindmjoshi** [comment]
old

**Pull request closed without merging**